geoalchemy2/functions.py

```diff
@@ -138,6 +167,10 @@ def __init__(self, *args, **kwargs):
     # Geometry Editors
     #
 
+    ('ST_AddPoint', types.Geometry,
+     'Adds a point to a LineString before point <position> (0-based index). '
+     'Third parameter can be omitted or set to -1 for appending.'),
+
```

Comment:
"Add a point to a LineString." is sufficient for the function docstring. See my comment above.